src/utilities/process-reply.js

```diff
@@ -38,7 +38,7 @@ module.exports = async function processReply({
         if (supportedLanguage) {
           const formattedCode = formatter(content, supportedLanguage);
           ref.channel.send(
-            formatMessageWithCodeblock(languageGuesses[0], formattedCode)
+            formatMessageWithCodeblock(supportedLanguage, formattedCode)
```

Comment:
If we are just using `supportedLanguage` now, what about the `languageGuesses` param? Does the bot no longer ever give multiple language suggestions?

Reply:
I've explained this details in the message below...